nox/sessions.py

```diff
@@ -224,75 +228,50 @@ def skip(self, *args, **kwargs):
         raise _SessionSkip()
 
 
-class Session(object):
+class SessionRunner(object):
     def __init__(self, name, signature, func, global_config, manifest=None):
         self.name = name
         self.signature = signature
         self.func = func
         self.global_config = global_config
         self.manifest = manifest
+        self.venv = None
 
     def __str__(self):
         return utils.coerce_str(self.signature or self.name)
```

Comment:
You presumably do not need `coerce_str` anymore, right? `str(self.signature or self.name)` should do the trick.

Reply:
Probably not, but I'll defer that for another PR for 2.7 cleanup.